07-multi-hop/07-multi-hop.md

```diff
@@ -1,12 +1,38 @@
 ## Goal: Check multi-hop connectivity over IPv6
 
+### Pre-setup Steps
+1. Book n>15 iotlab M3 nodes (preferably in grenoble) in a straight line 3 nodes apart. (so x--x--x--x)
```

Comment:
Why 15? The test specs ask for 5.

Reply:
This is because it is very hard to get proper positioning.  If I book a whole bunch I don't need to alter the position I can just move to the next node.

Reply:
From my experience ~7 nodes apart is enough to only have the neighboring nodes reachable. And in the end you still can use `l2filter_blacklist` to remove the nodes that aren't supposed to reach each other.

Reply:
The blacklisting seems like a good solution, especially for automation.  However, this is how I tested.  I found it was very hard to get the spacing correct and unpleasant to have to cancel and rebook an experiment because one node couldn't reach or another node could reach two.  It is not optimized but it worked.  It may also be good to test in other ways (a very controlled minimal way so we can confirm specs and a wide open way so we can see unpredictable but still valid behaviors).